### ISSUE TITLE
Feedback on Dyslexia poster

### ISSUE THREAD

**Issue by Hamtara** [UKHomeOffice/posters#107]
Hi,

Thank you for those resources, those poster are great. I would just like to point out that the font choice is also important for Dyslexic people. Indeed, the usage of Serif fonts make it hard for some people to read the content at a good pace.

Thank you.

**Comment by shawnthompson** [comment]
There was a discussion about avoid over use of _emphasis or italics_ also.

**Comment by Hamtara** [comment]
Indeed, italic and emphasis have also a big impact on readability. There is a good study available about [fonts for dyslexia](http://dyslexiahelp.umich.edu/sites/default/files/good_fonts_for_dyslexia_study.pdf).
